vital/types/track.h

```diff
@@ -285,6 +285,28 @@ class VITAL_EXPORT track : public std::enable_shared_from_this<track>
   /// Return whether or not this track has any states.
   bool empty() const { return history_.empty(); }
 
+  /**
+   * @brief Get attributes set.
```

Comment:
This is inconsistent Doxygen style for this file.  other functions use `///` for the brief comment and `\return` instead of `@return`.  As we many thing in KWIVER we have multiple conflicting styles.  We should pick one style and put in the work to make everything consistent with that style, but for now I think it's best to be at least consistent within the file.

Reply:
Ack... this is copied from `detected_object_set.h` and really ought to be kept in sync. Accordingly, I'd really prefer to not change it unless all of either `track.h` and/or `detected_object_set.h` is also being overhauled, which means we really ought to first agree on a preferred style.

Reply:
Has this been addressed or is it waiting for consensus on DoxyGen style?

Reply:
This has now been addressed. To maintain consistency, I also updated `detected_object_set.h`.